core/src/kvs/tx.rs

```diff
@@ -102,6 +102,8 @@ pub(super) enum Inner {
 	TiKV(super::tikv::Transaction),
 	#[cfg(feature = "kv-fdb")]
 	FoundationDB(super::fdb::Transaction),
+	#[cfg(feature = "kv-surrealkv")]
```

Comment:
What about `kv-surreal` ?

Reply:
I think `kv-surrealkv` is more consistent with our current naming scheme, `kv-{KV_STORE_NAME}`. This is regardless of the name of the KV store. I know the `kv` in `surrealkv` can feel a bit redundant given that the feature is prefixed by `kv-` but we also have `kv-tikv`, instead of `kv-ti` 😅

Reply:
I suppose we could also shorten it to `kv-skv`, similar to how we shorten FoundationDB's feature flag to `kv-fdb`.

Reply:
I don't have any strong opinions on this but I think keeping it explicit is better.